src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -237,10 +238,10 @@ export class ExposureNotificationService {
       return undefined;
     })();
 
-    const finalize = (status: ExposureStatus) => {
+    const finalize = async (status: ExposureStatus) => {
       const timestamp = `${new Date().getTime()}`;
       this.exposureStatus.set({...status, lastChecked: timestamp});
-      this.storage.setItem(LAST_CHECK_TIMESTAMP, timestamp);
+      await this.storage.setItem(LAST_CHECK_TIMESTAMP, timestamp);
```

Comment:
It's better wait for storage to write successfully before returning.

Reply:
why is that? we don't handle any failure cases here, waiting for the success seems pointless.